test/middlewares/request_options_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Middlewares.OptionsTest do
```

Comment:
Should follow file name and the tested module's name , so RequestOptionsTest

Reply:
👍